IOS/SwiftSeedProject/SwiftSeedProject/Application/SourceViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SourceViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 18/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import Bond
+
+class SourceViewModel {
+    
+    private let source: Source!
+    
+    let identifier: Observable<String>
+    let name: Observable<String>
+    let url: Observable<URL>
+    let detail: Observable<String>
+    let language: Observable<Language>
+    let country: Observable<Country>
+    let category: Observable<SourceCategory>
+    
+    init(source: Source) {
+        self.source = source
```

Comment:
Not necessary at all, we can remove it. The binded properties are the Observable ones.